### PR TITLE
5 new civilian base templates used in the UnitPlacer.

### PR DESCRIPTION
**Pull request by clivepaterson**

### CODE REVIEW

File: shared/src/main/resources/base_template/CybranBrick.lua
Comment (clivepaterson) [inline shared/src/main/resources/base_template/CybranBrick.lua:1]: (Old image from earlier commit)
image

File: shared/src/main/resources/base_template/CybranMantisBase.lua
Comment (clivepaterson) [inline shared/src/main/resources/base_template/CybranMantisBase.lua:1]: image

File: shared/src/main/resources/base_template/RadarAndPDBase.lua
Comment (clivepaterson) [inline shared/src/main/resources/base_template/RadarAndPDBase.lua:1]: (Old image from earlier commit)
image

File: shared/src/main/resources/base_template/UEFPegenBase.lua
Comment (clivepaterson) [inline shared/src/main/resources/base_template/UEFPegenBase.lua:1]: (Old image from earlier commit)
image

File: shared/src/main/resources/base_template/UEFTitans.lua
Comment (clivepaterson) [inline shared/src/main/resources/base_template/UEFTitans.lua:1]: (Old image from earlier commit)
image

**Comment by clivepaterson** [comment]
@BlackYps , any thoughts on these civ bases being added?

File: shared/src/main/resources/base_template/UEFTitans.lua
Comment (Sheikah45) [reply shared/src/main/resources/base_template/UEFTitans.lua:1]: This one just seems a little empty to me. Also the walls could probably use a small opening so things don't look completely trapped.

File: shared/src/main/resources/base_template/CybranBrick.lua
Comment (Sheikah45) [reply shared/src/main/resources/base_template/CybranBrick.lua:1]: Might be good to leave an opening in the walls.

Comment (clivepaterson) [reply shared/src/main/resources/base_template/CybranBrick.lua:1]: image

V2, this one is a rotated copy.

File: shared/src/main/resources/base_template/CybranMantisBase.lua
Comment (clivepaterson) [reply shared/src/main/resources/base_template/CybranMantisBase.lua:1]: image
Minor updates

File: shared/src/main/resources/base_template/RadarAndPDBase.lua
Comment (clivepaterson) [reply shared/src/main/resources/base_template/RadarAndPDBase.lua:1]: Minor updates
image

File: shared/src/main/resources/base_template/UEFPegenBase.lua
Comment (clivepaterson) [reply shared/src/main/resources/base_template/UEFPegenBase.lua:1]: image
Minor updates based on Storm and Index chats

File: shared/src/main/resources/base_template/UEFTitans.lua
Comment (clivepaterson) [reply shared/src/main/resources/base_template/UEFTitans.lua:1]: image
Updates based on feedback

**Review by Sheikah45 — APPROVED** [review]

**Pull request merged**